hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/server/namenode/ha/ConfiguredFailoverProxyProvider.java

```diff
@@ -93,4 +109,65 @@ public synchronized void close() throws IOException {
   public boolean useLogicalURI() {
     return true;
   }
+
+  /**
+   * Write active NameNode index to cache file.
+   */
+  public void writeActiveCache(int index) {
```

Comment:
doesn't need to be public. If it has to be public, please make sure to check that index is non-negative.

Reply:
OK, I'll correct it.

Reply:
Sorry, I have missed this comment before, Done now.